Soccar [Physics]/src/nl/soccar/physics/GameEngine.java

```diff
@@ -127,18 +130,58 @@ private void checkScored() {
         }
     }
 
+    public void addCar(Player player, CarPhysics car) {
+        cars.put(player, car);
+
+        synchronized (objects) {
+            objects.add(car);
+        }
+    }
+
+    public void removeCar(Player player) {

```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AUndocumentedApi)